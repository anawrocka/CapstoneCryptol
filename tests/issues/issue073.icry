1 : [(2^^37)-0x100]
let f x y = (x : [(2^^37)-(2^^5)]) + y
f zero zero