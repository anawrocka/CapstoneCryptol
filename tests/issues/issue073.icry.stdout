Loading module Cryptol

word too wide for memory: 137438953216 bits

word too wide for memory: 137438953440 bits
